### DISCUSSION TITLE
DataDocs management

### DISCUSSION THREAD

**Discussion by MichalMakaruk21** [great-expectations/great_expectations#6688] [Ideas]
**Is your feature request related to a problem? Please describe.**

I would like to know if there is plan to create simple tool to manage history of DataDocs. 

**Describe the solution you'd like**

- Possibility to delete misguided validations from DataDocs without deleting entire content.  
- Possibility to change the status from **failed** to **ignore** with the comment option.
- Divide DataDocs by final user, for example, **developer** and **client**.

**Additional context**

Solution should provide **local** and **cloud** management of DataDocs.

**Comment by austiezr** [comment]
Hey @MichalMakaruk21! Thanks for raising this; while this functionality is currently in progress for open source, we would love to continue this discourse with community around desired behaviors in Data Docs. To that end, I'm transferring this issue into Discussions.

Additionally, if you're looking for cloud management of Data Docs, you may be interested in one of our [free Cloud workshops](https://greatexpectations.io/gx-cloud).
